editor/src/components/canvas/canvas-strategies/strategies/drag-to-insert-metastrategy.tsx

```diff
@@ -343,8 +347,7 @@ function runTargetStrategiesForFreshlyInsertedElement(
   }>,
   strategyLifeCycle: InteractionLifecycle,
 ): Array<EditorStatePatch> {
-  const storyboard = MetadataUtils.getStoryboardMetadata(editorState.jsxMetadata)
-  const rootPath = storyboard != null ? storyboard.elementPath : elementPath([])
+  const rootPath = getRootPath(editorState.jsxMetadata) ?? EP.elementPath([])
```

Comment:
I know this is based on the previous code, but I'm not sure we should protect against there being no storyboard path here, because the empty element path fallback would be completely wrong. Maybe it's worth having this throw like you've done above?

Reply:
https://github.com/concrete-utopia/utopia/pull/3623/commits/90f446e0e1a2b21313327ee2c6cb0dfb339585f9